src/App.tsx

```diff
@@ -27,35 +34,101 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const LanguageSelector = (props: LanguageSelectorProps): ReactElement => {
-  const [name, setName] = useState(props.name);
-  const classes = useStyles();
-  const tStyle = { color: 'white' };
+const enumStyles = makeStyles((theme) => ({
```

Comment:
we don't need to have two of those since they're holding exactly the same content. Leave it as useStyle and just call useStyle where you were calling enumStyles.